client/test_contracts/test_groth16_altbn128_mixer_base.py

```diff
@@ -174,17 +174,17 @@ class TestGroth16AltBN128MixerBaseContract(TestCase):
 
```

Comment:
TODO: Rename this python function for consistency

Reply:
Also check the python code to make sure all python functions/classes' names align with the renamed solidity contracts